.stylelintrc

```diff
@@ -1,8 +1,8 @@
 plugins:
   - stylelint-scss
+customSyntax: postcss-scss
 rules:
   color-no-invalid-hex: true
-  function-calc-no-invalid: true
```

Comment:
This rule was removed without a replacement - see https://github.com/stylelint/stylelint/issues/5713